.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+jobs:
+  build:
+    macos:
+      xcode: "9.4.0"
+
+    steps:
+      - checkout
+
+      - run:
+          name: Update submodules
+          command: git submodule update --init --recursive
+
+      - run:
+          name: Build and run tests
+          command: fastlane scan
+          environment:
+            SCAN_DEVICE: iPhone 8
+            SCAN_SCHEME: libExpecta-iOS
+
+      - store_test_results:
+          path: test_output/report.xml
```

Comment:
Are you sure about this? According to `scan` documentation the default output directory is `./fastlane/test_output`. I see that `circle-ci` for `libextobjc` for example has the same path, but if I look at the output of a [job](https://circleci.com/gh/Lightricks/libextobjc/2) I see: 
```
Unable to save test results from /Users/distiller/project/test_output/report.xml
Error stat /Users/distiller/project/test_output/report.xml: no such file or directory
Found no path with test results, skipping
```